app/javascript/dashboard/components/ChatList.vue

```diff
@@ -246,6 +296,14 @@ export default {
 
       return conversationList;
     },
+    activeCustomView() {
+      if (this.customViewsId) {
+        return this.customViews.filter(
```

Comment:
This will be returning an array but else case is returning `{}` which is object. Please fix it

Reply:
Fixes by https://github.com/chatwoot/chatwoot/pull/3666/commits/e81e351b5bde30d9c283845aabadb8c26acf2680